Engine/Colliders/Collider.cs

```diff
@@ -0,0 +1,49 @@
+﻿using AntEngine.Maths;
+
+namespace AntEngine.Colliders
+{
+    /// <summary>
+    /// Abstract collider allows for the calculation of collision between primitive geometrical shapes.
+    /// </summary>
+    public abstract class Collider
+    {
+        /// <summary>
+        /// Boundaries of the collider.
+        /// </summary>
```

Comment:
This is not representing the boundaries, this is representing the position, rotation and scale of this collider.
What we may do is to just use the same transform as the entity or use this transform to apply a local transformation to the collider (this transform would be in the entity's referential)